docs/customizing_token_claims.rst

```diff
@@ -20,7 +20,7 @@ generated by the ``TokenObtainPairView``:
           token = super().get_token(user)
 
           # Add custom claims
-          token['name'] = user.name
```

Comment:
I'd prefer that we stick to the pattern of using the `__setitem__` implementation instead of doing things to `payload` directly.

Reply:
Any particular reason? If we wanted to make it private, we could do `__payload`, but sometimes it's necessary to work with the dict's methods.